components/10grid.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/docs.css">
+    <title>Even UI | Grid</title>
+    <script src="https://kit.fontawesome.com/9f366bbf8a.js" crossorigin="anonymous"></script>
+    <script src="./index.js"></script>
+</head>
+
+<body>
+    <nav class="topnav" id="myTopnav">
+        <p class="nav-itm nav-itm-text active "><a href="../index.html">Home</a> </p>
+        <p class="nav-itm right "> <span>Github</span> <i class="fab fa-github"></i></p>
+        <p class="nav-itm right "> <span>Repository</span> <i class="fas fa-code-branch"></i></i></p>
+        <p class="nav-itm control right" onclick="myFunction()" > <i class="fa fa-bars"></i></p>
```

Comment:
instead of using `onclick` try to use `document.getElementByClassName('control')[0].addEventListener('click',()=>{})` 

it would increase readability and decrease the confusion